### DISCUSSION TITLE
Cardinal Doesn't work

### DISCUSSION THREAD

**Discussion by Eluun** [DISTRHO/Cardinal#365] [General]
Hi everyone, i'm pretty excited to test this incredible vst but, unfortunaly, i was installed it on windows and added it on fl studio but, nothing happen.

I opened the synth on my channel rack, nothing.

Open Cardinal in my mixer with a "midi out" linked, nothing happen too. Same with CaridnalFx.

I dont understand why, i opened a preset, all is perfectly rooted .

The only think work is when i put a VCO or DCO directly on a host audio out, without ADSR. I think the problem is from the midi receving because when i put directly a VCO on out and play with my synth, the note doesn't change.

**Comment by subderground** [comment]
Hi,
You have to link midi infos from your Daw to CV,-Gates of cardinal vst. I don't use FL but Reaper.
But Cardinal works perfectly. I think it's a routing trick (maybe midi channel) inside FL.
Good luck.
